File: BookApp/src/main/java/com/benxinm/bookapp/entity/Book.java
package com.benxinm.bookapp.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Book {
    private Integer id;
    private String name;
    private String author;
    private String remark;
    private String coverRef;
}

File: NoevlApplication/app/src/main/java/com/benxinm/novelapplication/bean/Book.kt
package com.benxinm.novelapplication.bean

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
class Book(val id:String,val name:String,val author:String,val remark:String,val coverRef:String): Parcelable {
}